data_steward/tools/copy_dataset_to_output_prod.py

```diff
@@ -160,16 +160,19 @@ def get_arg_parser() -> argparse.ArgumentParser:
         f'Copying fitbit tables from dataset {args.src_project_id}.{args.fitbit_dataset_id} to {args.src_project_id}.{args.src_dataset_id}...'
     )
 
-    bq_client.copy_dataset(f'{args.src_project_id}.{args.fitbit_dataset_id}',
-                           f'{args.src_project_id}.{args.src_dataset_id}')
+    copy_fitbit_jobs = bq_client.copy_dataset(
+        f'{args.src_project_id}.{args.fitbit_dataset_id}',
+        f'{args.src_project_id}.{args.src_dataset_id}')
+    bq_client.wait_on_jobs(copy_fitbit_jobs)
```

Comment:
It would be nice if the wait happened inside the `copy_dataset` method.  That way no one has to remember to add this additional line each time we want to use the `copy_dataset` method.

Reply:
Good point, will update it.